zone/spell_effects.cpp

```diff
@@ -1331,7 +1331,7 @@ bool Mob::SpellEffect(Mob* caster, uint16 spell_id, float partial, int level_ove
 				snprintf(effect_desc, _EDLEN, "Blind: %+i", effect_value);
 #endif
 				// 'cure blind'
-				if (BeneficialSpell(spell_id) && spells[spell_id].buff_duration == 0) {
+				if (IsDistanceModifierSpell(spell_id) && spells[spell_id].buff_duration == 0) {
```

Comment:
Going from BeneficialSpell to IsDistanceModifierSpell is this expected?

Reply:
Accident with find and replace. I’ll fix it. Good eye.